doc/source/example-newsreader.rst

```diff
@@ -1,29 +1,96 @@
 News Reader
 ===========
 
-This document shows how to implement a simple news reader using Ray. The reader
-consists of a simple Vue.js `frontend`_ and a backend consisting of a Flask
-server and a Ray actor. View the `code for this example`_.
+This document walks you through implementing a simple news reader using Ray.
+The reader consists of a simple Vue.js frontend and a Ray backend.
+It has a simple recommendation engine built in, which you will train on upvotes
+from hackernews and which learns online as the user stars articles.
+
+It looks like this (the frontend is based on `QReader`_), here the recommended
+articles are in yellow:
+
+.. image:: newsreader.jpg
+
+View the `code for this example`_.
 
 To run this example, you will need to install NPM and a few python dependencies.
 
 .. code-block:: bash
 
   pip install atoma
   pip install flask
+  pip install vowpalwabbit
```

Comment:
`pip install vowpalwabbit` fails for me with `Exception: Could not find boost python library`